straxen/plugins/led_cal/led_calibration.py

```diff
@@ -55,21 +56,23 @@ class LEDCalibration(strax.Plugin):
         help="Window (samples) for baseline calculation.",
     )
 
-    default_led_window = straxen.URLConfig(
-        default=(78, 107),
+    minimum_led_position = straxen.URLConfig(
+        default=60,
         infer_type=False,
-        help="Default window (samples) to integrate and get the maximum amplitude \
-            if no hit was found in the record.",
+        help=(
+            "The minimum sample index to consider for LED hits. Hits before this sample are"
```

Comment:
`"The minimum sample index to consider for LED hits. Hits before this sample are"` -> `"The minimum sample index to consider for LED hits. Hits before this sample are "`

Reply:
fixed